src/tests/ftest/telemetry/pool_space_metrics.py

```diff
@@ -0,0 +1,178 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from ior_test_base import IorTestBase
+from telemetry_test_base import TestWithTelemetry
+
+# Pretty print constant
+RESULT_OK = "[\033[32m OK\033[0m]"
+RESULT_NOK = "[\033[31mNOK\033[0m]"
+
+# It should take as much as 10s for the vos space metrics to be up to date: it is equal to the idle
+# GC ULT interval time.
+TIMEOUT_DEADLINE = 15
+
+
+class TelemetryPoolSpaceMetrics(IorTestBase, TestWithTelemetry):
+    # pylint: disable=too-many-ancestors
```

Comment:
This shouldn't be needed anymore. See #11800

Reply:
Fixed with commit eb553bde2b1e538977535fb9da04a1eefb6b8dd8